substrate/bin/node/runtime/src/impls.rs

```diff
@@ -240,13 +240,60 @@ impl InstanceFilter<RuntimeCall> for NftsCallFilter {
 	Clone, Eq, PartialEq, Encode, Decode, RuntimeDebug, MaxEncodedLen, scale_info::TypeInfo, Serialize, Deserialize,
 )]
 pub struct CouncilAccountCallFilter;
+
+impl CouncilAccountCallFilter {
+	fn ensure_schedule(c: &RuntimeCall) -> Result<(&RuntimeCall, &BlockNumber), ()> {
+		if let RuntimeCall::Scheduler(pallet_scheduler::Call::schedule { when, call, ..}) = c {
+			Ok((call, when))
+		} else if let RuntimeCall::Scheduler(pallet_scheduler::Call::schedule_named { when, call, ..}) = c {
+			Ok((call, when))
+		} else {
+			Err(())
+		}
+	}
+
+	fn ensure_batched(c: &RuntimeCall) -> Result<&Vec<RuntimeCall>, ()> {
+		if let RuntimeCall::Utility(pallet_utility::Call::batch { calls }) = c {
+			Ok(calls)
+		} else if let RuntimeCall::Utility(pallet_utility::Call::batch_all { calls }) = c {
+			Ok(calls)
+		} else {
+			Err(())
+		}
+	}
+
+	fn ensure_transfer_or_remark(c: &RuntimeCall) -> Result<(), ()> {
+		match c {
+			RuntimeCall::LLM(pallet_llm::Call::remark { .. }) => Ok(()),
+			RuntimeCall::LLM(pallet_llm::Call::send_llm { .. }) => Ok(()),
+			RuntimeCall::LLM(pallet_llm::Call::send_llm_to_politipool { .. }) => Ok(()),
+			RuntimeCall::Assets(pallet_assets::Call::transfer { .. }) => Ok(()),
+			RuntimeCall::Assets(pallet_assets::Call::transfer_keep_alive { .. }) => Ok(()),
+			RuntimeCall::Balances(pallet_balances::Call::transfer { .. }) => Ok(()),
+			RuntimeCall::Balances(pallet_balances::Call::transfer_keep_alive { .. }) => Ok(()),
+			_ => Err(())
+		}
+	}
+
+	fn ensure_valid_council_call(c: &RuntimeCall) -> Result<(), ()> {
+		let (scheduled_call, when) = Self::ensure_schedule(c)?;
+		if when - System::block_number() <= 4 * DAYS {
```

Comment:
This is where we decide on the minimum delay. I remember talks about 7 days, but see some chatter that it should actually be 4 days. Please confirm the expected value.

Reply:
4 is ok